src/button/test/id.test.ts

```diff
@@ -1,53 +1,28 @@
 import { expect, fixture } from "@open-wc/testing";
 import { Button } from "../index";
 
-describe("id default prop is null", () => {
-  const container = new Button({});
+describe("Button", () => {
+  describe("id", () => {
+    it('should be "" when not assigning on constructor', async () => {
+      const container = new Button({});
 
-  it("id default prop is null", async () => {
-    const el = await fixture(container);
-    await expect(el.id).to.be.equals("");
-  });
-});
+      const el = await fixture(container);
+      expect(el.id).to.be.equals("");
+    });
 
-describe("id default prop set successfully", () => {
-  const container = new Button({ id: "options-id" });
-
-  it("id default prop set successfully", async () => {
-    const el = await fixture(container);
-    await expect(el.id).to.have.equals("options-id");
-  });
-});
-
-describe("id prop replace successfully", () => {
-  const container = new Button({
-    id: "options-id"
-  });
-  container.id = "replace-id";
+    it('should be "options-id" when assigning on constructor', async () => {
+      const container = new Button({ id: "options-id" });
 
-  it("id prop replace successfully", async () => {
-    const el = await fixture(container);
-    await expect(el.id).to.have.equals("replace-id");
-  });
-});
-
-describe("id default prop set to null", () => {
-  // @ts-ignore
-  const container = new Button({ id: null });
-
-  it("id default prop set to null", async () => {
-    const el = await fixture(container);
-    await expect(el.id).to.be.equals("null");
-  });
-});
+      const el = await fixture(container);
+      expect(el.id).to.have.equals("options-id");
```

Comment:
uses to.have.equal or to.have.string

Reply:
@nmanhit 
Id will be fixed string and i will using ".to.equal" instead of ".to.have.equals"